src/components/Article.jsx

```diff
@@ -0,0 +1,51 @@
+import { useNavigate } from "react-router-dom";
+
+const Article = ({ article }) => {
+  const {
+    title,
+    author,
+    article_id,
+    body,
+    topic,
+    created_at,
+    votes,
+    article_img_url,
+    comment_count,
+  } = article;
+
+  const navigate = useNavigate();
+
+  const date = new Date(created_at);
+  const min = date.getMinutes();
+  const hr = date.getHours();
+  const dd = date.getDate();
+  const yy = date.getFullYear();
+  const mm = date.getMonth() + 1;
+  const dateStr = `${min}:${hr} ${dd}/${mm}/${yy}`;
+
+  const handleClick = () => {
+    navigate(-1);
+  };
```

Comment:
This does work, but how else can we navigate to a different page using react router?